.github/workflows/update-reachability-metadata-version.yml

```diff
@@ -0,0 +1,67 @@
+name: "Create scheduled reachability metadata version updates"
+
+on:
+  # Reachability metadata releases happen on 1st and 15th of each month -> we should look for updates the day after
+  schedule:
+    - cron: "0 0 2 * *"
+    - cron: "0 0 16 * *"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  actions: write
+
+jobs:
+  get-latest-release-version:
+    runs-on: ubuntu-latest
+    outputs:
+      new_version: ${{ steps.get_release.outputs.new_version }}
+    steps:
+      - name: Get latest release tag
+        id: get_release
+        run: |
+          tag=$(curl -s https://api.github.com/repos/oracle/graalvm-reachability-metadata/releases/latest | jq -r .tag_name)
+          echo "new_version=$tag" >> $GITHUB_OUTPUT
+
+  get-currently-used-version:
+    runs-on: ubuntu-latest
+    needs: get-latest-release-version
+    outputs:
+      old_version: ${{ steps.get_current_version.outputs.old_version }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get current metadataRepository version
+        id: get_current_version
+        run: |
+          value=$(grep '^metadataRepository\s*=' gradle/libs.versions.toml | sed -E 's/.*=\s*"(.*)"/\1/')
+          echo "old_version=$value" >> $GITHUB_OUTPUT
+
+  update-version-and-create-pull-request:
+    runs-on: ubuntu-latest
+    needs: [get-latest-release-version, get-currently-used-version]
```

Comment:
Do you really need three jobs for this? Can't this be done in just one?

Reply:
Personally, this looks more readable to me. I split the code into three logically independent chunks, with clear responsibilities for each of them. I could put everything in one job, but in my opinion, that would make the code harder to read without providing significant benefits (unless I am missing some advantages of having all the steps in one job).